src/main/java/com/internet/shop/controllers/order/GetAllOrdersOfUserAdminController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controllers.order;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.service.interfaces.OrderService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllOrdersOfUserAdminController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllOrdersAdminController extends HttpServlet {
```
Admin should be able to see all orders in our shop

Reply:
ok, I did it for a specific user a specially but fixed it to all